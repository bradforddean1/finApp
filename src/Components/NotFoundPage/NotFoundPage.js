import React from "react";
import PageError from "../Common/PageError/PageError";

/**
 * 404
 * @component
 */
function NotFoundPage(props) {
	return (
		<PageError
			// className="ApplicationError"
			title="404"
			content="Content not found."
		></PageError>
	);
}

export default NotFoundPage;
