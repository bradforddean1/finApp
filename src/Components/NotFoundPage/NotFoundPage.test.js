import React from "react";
import ReactDOM from "react-dom";
import NotFoundPage from "./NotFoundPage";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("NotFoundPage Component", () => {
	it("Renders", () => {
		const div = document.createElement("div");
		ReactDOM.render(<NotFoundPage />, div);
		ReactDOM.unmountComponentAtNode(div);
	});

	it("Renders the Default State", () => {
		const wrapper = shallow(<NotFoundPage />);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
