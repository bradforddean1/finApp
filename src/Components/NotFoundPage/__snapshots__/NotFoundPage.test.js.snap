// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Component Renders the Default State 1`] = `
<PageError
  content="Content not found."
  title="404"
/>
`;
